src/PrestaShopBundle/Resources/config/services/core/grid/grid_factory.yml

```diff
@@ -143,6 +143,8 @@ services:
       - '@prestashop.core.grid.factory.language_decorator'
       - '@prestashop.core.grid.filter.form_factory'
       - '@prestashop.core.hook.dispatcher'
+    tags:
+      - { name: 'prestashop.grid_factory', grid: 'language' }
```

Comment:
it would be nice if here the "language" would be somehow forced to be come the GRID_ID from the LanguageGridDefinitionFactory, so we don't have overlapping ids of different purposes, but idk if thats doable? (anyway maybe you have some other ideas to improve that part if you get what i mean?)

Reply:
The compiler pass is already checking overlaps.

btw I thought about using the service id as a identifier, but there are some limitations.

It is easy for services like 
- `prestashop.core.grid.factory.meta => meta`
- `prestashop.core.grid.factory.category => category`

But complicated for services like:
- `prestashop.core.grid.factory.customer.address`, the name contains a `.` 

That would lead to services with `_` and others with `.`. Not sure if this is what we want